src/main/java/aztech/modern_industrialization/compat/kubejs/nuclear/CreateIsotopeFuelParamsEventJS.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Azercoco & Technici4n
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package aztech.modern_industrialization.compat.kubejs.nuclear;
+
+import aztech.modern_industrialization.nuclear.NuclearConstant;
+import dev.latvian.mods.kubejs.event.EventJS;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+public class CreateIsotopeFuelParamsEventJS extends EventJS {
+
+    public static SortedMap<String, NuclearConstant.IsotopeFuelParams> FUEL_PARAMS = new TreeMap<>();
+
+    public void createIsotopeFuelParams(String name, double thermalAbsorbProba, double thermalScatterings, int maxTemp, int tempLimitLow,
```

Comment:
I think that these events are a bit of a roundabout way to go about things. My intuition is that the registration of nuclear components can probably be simplified a little to make this unnecessary? Not sure

Reply:
Yeah, I thought about that as well. I couldn't find a way to make it less complicated, but I'll try and come up with a better way to do things